php-nginx/7.0/etc/confd/conf.d/nginx_site_phpfpm.conf.toml

```diff
@@ -0,0 +1,6 @@
+[template]
+src   = "nginx/site_base.conf.tmpl"
+dest  = "/etc/nginx/sites-available/default-30-rewriteapp.conf"
```

Comment:
ok, fixed and giving expected output. Not much I can do with external testing for some of the env vars without the PR to generate self signed cert